packages/nextjs/app/builders/0xaa4C60b784E2b3E485035399bF1b1aBDeD66A60f/page.tsx

```diff
@@ -0,0 +1,56 @@
+import { GithubIcon } from "./_components/GithubIcon";
+import { WalletIcon } from "./_components/WalletIcon";
+import { XIcon } from "./_components/XIcon";
+import type { NextPage } from "next";
+import { BlockieAvatar } from "~~/components/scaffold-eth";
+import { getMetadata } from "~~/utils/scaffold-eth/getMetadata";
+
+export const metadata = getMetadata({
+  title: "Danitome24 builder page",
+  description: "Builder page for danitome24 with a little bio and social links.",
+});
+
+const Builder: NextPage = () => {
```

Comment:
Since there will be more builder pages, you could consider changing ```Builder``` in a more descriptive way, like ```Danitome24Page```

Reply:
Done!